sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/MapsSearchAsyncClient.java

```diff
@@ -95,2234 +60,923 @@
-     * <pre>
-     * Response&lt;SearchAddressResult&gt; fuzzySearchResponse = asyncClient.fuzzySearchWithResponse&#40;
-     *     new FuzzySearchOptions&#40;&quot;Monaco&quot;&#41;.setEntityType&#40;GeographicEntityType.COUNTRY&#41;
-     *         .setTop&#40;5&#41;&#41;.block&#40;&#41;;
-     * String fuzzySearchId = fuzzySearchResponse.getValue&#40;&#41;.getResults&#40;&#41;.get&#40;0&#41;.getDataSource&#40;&#41;.getGeometry&#40;&#41;;
-     * List&lt;String&gt; getPolygonIds = results.getResults&#40;&#41;.stream&#40;&#41;
-     *     .filter&#40;item -&gt; item.getDataSource&#40;&#41; != null &amp;&amp; item.getDataSource&#40;&#41;.getGeometry&#40;&#41; != null&#41;
-     *     .map&#40;item -&gt; item.getDataSource&#40;&#41;.getGeometry&#40;&#41;&#41;
-     *     .collect&#40;Collectors.toList&#40;&#41;&#41;;
-     * getPolygonIds.add&#40;fuzzySearchId&#41;;
-     *
-     * if &#40;ids != null &amp;&amp; !getPolygonIds.isEmpty&#40;&#41;&#41; &#123;
-     *     System.out.println&#40;&quot;Get Polygon: &quot; + ids&#41;;
-     * &#125;
-     * </pre>
-     * <!-- end com.azure.maps.search.async.get_polygon -->
-     *
-     * @param geometryIds Comma separated list of geometry UUIDs, previously retrieved from an Online Search request.
+     * Use to get polygon data of a geographical area shape such as a city or a country region.
+     *
+     *
+     * The `Get Polygon` API is an HTTP `GET` request that supplies polygon data of a geographical area outline such as
+     * a city or a country region.
+     *
+     * @param coordinates A point on the earth specified as a longitude and latitude. Example: &amp;coordinates=lon,lat.
+     * @param view A string that represents an [ISO 3166-1 Alpha-2 region/country
+     * code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2). This will alter Geopolitical disputed borders and labels
+     * to align with the specified user region. By default, the View parameter is set to “Auto” even if you haven’t
+     * defined it in the request.
+     *
+     * Please refer to [Supported Views](https://aka.ms/AzureMapsLocalizationViews) for details and to see the available
+     * Views.
+     * @param resultType The geopolitical concept to return a boundary for. If not specified, the default is
+     * `countryRegion` result type.
+     * @param resolution Resolution determines the amount of points to send back. If not specified, the default is
+     * medium resolution.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ErrorResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return this object is returned from a successful Search Polygon call.
+     * @return `GeoJSON GeocodingFeature` object that describe the boundaries of a geographical area.
```

Comment:
Javadoc is HTML, this won't generate as a code reference like a Markdown file would. In this spot you need to either use `{@link ClassName}` or `{@code code}`. `@link` will create a hyperlink in the generated Javadoc that will link to the class (and you can link to a specific method or constant by using `classname#methodname`) and `@code` will create stylized text that represents generic code.